protocol.md

```diff
@@ -96,6 +96,23 @@ Steps to register a new *device*:
 
 Note that this procedure can only create *physical devices*, not *virtual ones*.
 
+### Session Tokens
+
+Prerequisite: The user is registered on the application.
+
+A *session token* can be obtained when a user creates their first device, or at any time later when proving ownership of a  [Verification Method](#verification-methods).
+
+Steps to obtain and check a *session token*:
+1. The *user* uses one the [Verification Method]s available to them, and requests a *session token* through the API
+1. *Tanker Core* generates a random nonce and sends it to the *Tanker server* along with the verification
```

Comment:
do nounce are not random by definition ?

Reply:
Not necessarily as long as they're only used once, but I can remove the word "random"